cassandane/Cassandane/Unit/Runner.pm

```diff
@@ -40,26 +40,28 @@
 package Cassandane::Unit::Runner;
 use strict;
 use warnings;
-use base qw(Test::Unit::TestRunner);
+use base qw(Test::Unit::Runner);
```

Comment:
This change is kind of weird to me.  This class is called Runner, but used to inherit from TestRunner… which itself inherits from Runner.  The docs are not great here.

I think the deal is that a TestRunner is a more concrete kind of runner, and it used to be that our (Cass's) Runner was just specializing that.  Now that we're building in this multiple formatter jobby, we're shifing our parent one level up toward the abstract.  Fine!

Actually, even better, because we're no longer inhering from something and then changing the signature on its constructor, so I'll sleep easier!

Reply:
Oh yeah this whole thing is a mess, I've slightly vacuumed one corner of one rug at best.  And yeah your interpretation of what's going on here is correct, Cass's Runner has become more abstract.  Specialising TestRunner used to get us output formatting (which we then mostly overrode anyway).  Now our FormatFoo's take care of output formatting, and Runner can just be a Runner (albeit one that needs to know about FormatFoos, because the T:U:Runner doesn't)